code/modules/mob/living/simple_animal/friendly/drone/_drone.dm

```diff
@@ -31,7 +31,8 @@
 	ventcrawler = VENTCRAWLER_ALWAYS
 	healable = 0
 	density = FALSE
-	pass_flags = PASSTABLE | PASSMOB
+	pass_flags = PASSTABLE | PASSMOB | PASSDOOR //yogs - allowed drones to pass thru airlocks again :D
+	light_color = "#E42742" //yogs - A hacked red, for when it needs to glow when hack'rd
```

Comment:
I'm pretty sure you can replicate these both to a file in the ``yogstation`` folder.